doc/ug/getting_started_fpga.md

```diff
@@ -61,6 +61,12 @@ The `fileset_top` flag used above is specific to the OpenTitan project to select
 The resulting bitstream is located at `build/lowrisc_systems_chip_earlgrey_cw310_0.1/synth-vivado/lowrisc_systems_chip_earlgrey_cw310_0.1.bit`.
 See the [reference manual]({{< relref "ref_manual_fpga.md" >}}) for more information.
 
+### Dealing with FPGA Congestion Issues
+
+The default vivado tool placement may sometimes result in congested FPGA floorplans.
```

Comment:
You may want to say that the congestion can lead to unpredictable and long implementation times and results. This is what the user might experience and how he/she realizes that something is going wrong.

Reply:
Nit: want to capitalize Vivado?

Reply:
updated!